api/models/article-models.js

```diff
@@ -66,3 +66,21 @@ exports.fetchArticleComments = (article_id, order) => {
     }
   });
 };
+
+exports.setArticleComment = (article, username, comment) => {
+  const promiseCatcher = [
```

Comment:
I'm not sure what promiseCatcher and valueCatcher mean. There may be better names you could choose for these variables?